library_generation/utilities.py

```diff
@@ -483,3 +499,52 @@ def get_version_from(
         for line in f.readlines():
             if artifact_id in line:
                 return line.split(":")[index].strip()
+
+
+def get_file_paths(config: GenerationConfig) -> set[str]:
+    """
+    Get versioned proto_path from configuration file, plus known paths of
+    common protos.
+
+    :param config:
+    :return: versioned proto_path plus paths of common protos
+    """
+    paths = set()
+    for library in config.libraries:
+        for gapic_config in library.gapic_configs:
+            paths.add(gapic_config.proto_path)
+    return paths.union(common_protos)
```

Comment:
I don't think we should include common protos, unless the common protos are included in google-cloud-java, the commit history of common protos should go into sdk-platform-java as part of java-common-protos changes.

Reply:
Done.